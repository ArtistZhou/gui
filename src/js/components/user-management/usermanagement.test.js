import React from 'react';
import renderer from 'react-test-renderer';
import UserManagement from './usermanagement';

it('renders correctly', () => {
  const tree = renderer.create(<UserManagement />).toJSON();
  expect(tree).toMatchSnapshot();
});
