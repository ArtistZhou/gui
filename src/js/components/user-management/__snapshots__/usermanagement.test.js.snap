// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="float-right"
  >
    <button
      className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-contained-12 MuiButton-containedPrimary-13 MuiButton-raised-15 MuiButton-raisedPrimary-16"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onContextMenu={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex="0"
      type="button"
    >
      <span
        className="MuiButton-label-2"
      >
        Create new user
      </span>
      <span
        className="MuiTouchRipple-root-73"
      />
    </button>
  </div>
  <div
    className="margin-top-small"
  >
    <div
      style={
        Object {
          "marginLeft": "20px",
        }
      }
    >
      <h2
        style={
          Object {
            "marginTop": "15px",
          }
        }
      >
        Users
      </h2>
    </div>
    <div
      className="margin-bottom"
    >
      <table
        className="MuiTable-root-30"
      >
        <thead
          className="MuiTableHead-root-31"
        >
          <tr
            className="MuiTableRow-root-32 MuiTableRow-head-35"
          >
            <th
              aria-sort={null}
              className="MuiTableCell-root-37 MuiTableCell-head-38"
              scope="col"
            >
              Email
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-37 MuiTableCell-head-38"
              scope="col"
            >
              Date created
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-37 MuiTableCell-head-38"
              scope="col"
            >
              Last updated
            </th>
            <th
              aria-sort={null}
              className="MuiTableCell-root-37 MuiTableCell-head-38"
              scope="col"
            >
              Manage
            </th>
          </tr>
        </thead>
        <tbody
          className="MuiTableBody-root-49"
        />
      </table>
    </div>
  </div>
</div>
`;
